### PR TITLE
setup cython linting

### PR DESCRIPTION
**Pull request by incognitojam**

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
Can we setup a linter for Cython?

**Comment by incognitojam** [comment]
can try adding this to precommit: https://github.com/MarcoGorelli/cython-lint

File: can/parser_pyx.pyx [hunk 1]
```diff
@@ -106,7 +104,7 @@ cdef class CANParser:
 
   def update_strings(self, strings, sendcan=False):
     for v in self.vl_all.values():
-      for l in v.values():
+      for l in v.values():  # no-cython-lint
```
Comment (incognitojam) [inline can/parser_pyx.pyx:107]: not sure what `vl` is, but cython complained about the `l` variable name

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+name = "opendbc"
+license = "MIT"
+readme = "README.md"
+repository = "https://github.com/commaai/opendbc"
+
+[tool.cython-lint]
+max-line-length = 120
+ignore = ["E111", "E114"]
```
Comment (incognitojam) [inline pyproject.toml:8]: these are for 4-space indentation

**Pull request merged**